seahorn/lib/klee_switch.c

```diff
@@ -53,6 +53,4052 @@ size_t alloc_size(size_t sz) {
     case 48: return 48;
     case 49: return 49;
     case 50: return 50;
+    case 51: return 51;
```

Comment:
ask @priyasiddharth to help implementing this in C++ so that there is no need to generate the file with python